asf_welsh_energy_consultation/pipeline/plotting.py

```diff
@@ -71,6 +71,14 @@ def time_series_comparison(
     if domain_max is None:
         domain_max = data.date.max()
         logger.info(f"Time series comparison using {domain_max} as max date")
+
+    # If single variable, remove legend
```

Comment:
you mean single category variables? right? 
```suggestion
    # If single variable category, remove legend
```

Reply:
Yes, this means that if a variable only has one category, the legend will be removed because it's not necessary.